src/main/resources/l10n/JabRef_in.properties

```diff
@@ -2295,8 +2295,15 @@ Select_first_entry=
 Select_last_entry=
 
 Invalid_ISBN\:_'%0'.=
+
+Hide_field=
+Optionals_fields=
+Show_Field=
 should_be_an_integer_or_normalized=
 should_be_normalized=
 Copy_BibTeX_key_and_link=
 empty_BibTeX_key=
 BibLaTeX_field_only=
+Driver_error=
+The_BibEntry_you_currently_work_on_has_been_deleted_on_the_shared_side._Hit_"Keep"_to_recover_the_entry=
```

Comment:
This is not in the language files of this PR. Please fix translations. Proposal: Use the translation files of the master branch and readd requried translations.